### PR TITLE
fix: check if protocol exists

### PR DESCRIPTION
**Pull request by mahdyar**
GitHub lets users input their profile URLs with or without the protocol, so if someone like me has a URL like `mahdyar.me` its link would go to `https://gitemon.netlify.app/<organization>/mahdyar.me` instead of `http(s)://mahdyar.me`.

### CODE REVIEW

**Comment by mahdyar** [comment]
@eduardoboucas Any thoughts?

File: src/Random.js [hunk 1]
```diff
@@ -235,6 +235,11 @@ function Random({ isAuthenticated, org, orgSlug, person }) {
     }
 
     if (hint.type === HINT_SITE) {
+      if (
+        person.blog.indexOf("http://") != 0 &&
+        person.blog.indexOf("https://") != 0
+      )
+        person.blog = "http://" + person.blog;
```
Comment (eduardoboucas) [inline src/Random.js:242]: I think in this case we should default to `https://` instead of `http://`.

**Review by eduardoboucas — CHANGES_REQUESTED** [review]
Sorry for the late reply! This looks good. I left a minor comment.

**Comment by mahdyar** [comment]
I've deleted the fork for this PR. so I'm gonna close this one and make a new one.

**Pull request closed without merging**